### PR TITLE
Metastore vfsplugin

### PR DESCRIPTION
**Pull request by abryant-hv**

### CODE REVIEW

**Comment by buildguy** [comment]
### :warning: Build finished in 41m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.di.job.entries.http.JobEntryHTTPTest.testExecute_simpleConfiguration</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.di.trans.TransTest.testPDI12424ParametersFromMetaAreCopiedToTrans</b> (click to expand)</summary><p>

```
Could not write to "file:///tmp/parameters_73e852c8-266d-11ee-81bc-71b13736ebc2.ktr".
```

</p></details>

  
Tests run: **5339**, Failures: **2**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-8957/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: engine/src/main/java/org/pentaho/di/core/util/ConnectionUtil.java [hunk 1]
```diff
@@ -24,10 +24,8 @@
 
 import org.pentaho.di.base.AbstractMeta;
 import org.pentaho.di.connections.ConnectionManager;
+import org.pentaho.di.core.osgi.api.MetastoreLocatorOsgi;
```
Comment (peterrinehart) [inline engine/src/main/java/org/pentaho/di/core/util/ConnectionUtil.java:27]: Is this import actually being used?

**Review by peterrinehart — CHANGES_REQUESTED** [review]

File: engine/src/main/java/org/pentaho/di/core/util/ConnectionUtil.java (hunk 1 above)
Comment (abryant-hv) [reply engine/src/main/java/org/pentaho/di/core/util/ConnectionUtil.java:27]: Good catch. Looks like it was used in an interim commit. Will remove.

**Comment by buildguy** [comment]
### :warning: Build finished in 41m 44s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.di.job.entries.http.JobEntryHTTPTest.testExecute_simpleConfiguration</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.di.trans.TransTest.testPDI12424ParametersFromMetaAreCopiedToTrans</b> (click to expand)</summary><p>

```
Could not write to "file:///tmp/parameters_de1ecb2b-2a59-11ee-a8af-ffdd8df5ec56.ktr".
```

</p></details>

  
Tests run: **5339**, Failures: **2**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-8957/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by peterrinehart — APPROVED** [review]

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 11m 49s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.di.job.entries.http.JobEntryHTTPTest.testExecute_simpleConfiguration</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.di.trans.TransTest.testPDI12424ParametersFromMetaAreCopiedToTrans</b> (click to expand)</summary><p>

```
Could not write to "file:///tmp/parameters_0adef5d5-2d58-11ee-85a0-e1e0f657a6aa.ktr".
```

</p></details>

  
Tests run: **5339**, Failures: **2**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-8957/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**